aiperf/gpu_telemetry/telemetry_manager.py

```
@@ -0,0 +1,235 @@
+        super().__init__(
+            service_config=service_config,
+            user_config=user_config,
+            service_id=service_id,
+        )
+
+        self._collectors: dict[str, TelemetryDataCollector] = {}
+
+        user_endpoints = user_config.server_metrics_url or []
+
+        if DEFAULT_DCGM_ENDPOINT not in user_endpoints:
+            self._dcgm_endpoints = [DEFAULT_DCGM_ENDPOINT] + user_endpoints
```

Ruff (0.13.1)

Consider `[DEFAULT_DCGM_ENDPOINT, *user_endpoints]` instead of concatenation

Replace with `[DEFAULT_DCGM_ENDPOINT, *user_endpoints]`

(RUF005)